net/tests/Sails.Remoting.Tests/_Infra/XUnit/TestFramework.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Xunit.Abstractions;
+
+[assembly: Xunit.TestFramework(
+    "Sails.Remoting.Tests._Infra.XUnit.TestFramework",
+    "Sails.Remoting.Tests")]
+
+namespace Sails.Remoting.Tests._Infra.XUnit;
```

Comment:
`Sails.Remoting.Tests.Internal.XUnit`

Reply:
I use _ on purpose in order to avoid any possible collisitions with namespaces from the project under test, i.e. in some regular project I might want to have `Internal` as a regular namespace